app/routines/update_user_salesforce_info.rb

```diff
@@ -191,6 +191,13 @@ def cache_contact_data_in_user!(contact, user)
 
       user.faculty_status = case contact.faculty_verified
       when "Confirmed"
+        SecurityLog.create!(
+            user: user,
+            application: nil,
+            remote_ip: nil,
+            event_type: :faculty_verified,
+            event_data: { user_id: user.id, salesforce_id: contact.id }
```

Comment:
@Dantemss - what do you think about also having the Salesforce ID, so you can easily go look them up if needed?

Reply:
That would be great. The more info you can provide the better IMO.

Reply:
Cool - so does this look ready @Dantemss?